libzerocash/Address.h

```diff
@@ -12,6 +12,20 @@
 #ifndef ADDRESS_H_
 #define ADDRESS_H_
 
+#include <cryptopp/osrng.h>
```

Comment:
This is necessary (it's the psuedorandom number generator which the `Decryptor` apparently needs) it's just that placing it in `Address.h` could be sketchy. I will relocate them to `Coin.cpp` to minimize fallout.